app/models/transaction.rb

```diff
@@ -1,11 +1,13 @@
 class Transaction < ApplicationRecord
-
   validates :invoice, presence: true
   validates :credit_card_number, presence: true
   validates :result, presence: true
 
   belongs_to :invoice
 
-  enum status: [failed: 0, success: 1]
-
+  def self.random
```

Comment:
will all of your models have this method? if so, could we put it in `ApplicationRecord` to help DRY our models up?